pkg/generator/key/key.go

```diff
@@ -14,4 +14,5 @@ const (
 
 var (
 	ConfigVersion = project.Name() + ".giantswarm.io/config-version"
+	AppLabel      = project.Name() + ".giantswarm.io/app"
```

Comment:
As discussed in standup this needs to be fixed by using labels from apiextensions. I'd go with `app.kubernetes.io/name` and `giantswarm.io/managed-by` that should make it clear what app it belongs to and what created it.